src/common.css

```diff
@@ -17,6 +17,15 @@
   top: 0px !important;
 }
 
+/* send offscreen without breaking sizes*/
+.ui-select-detached {
+    position: fixed !important;
+    opacity: 0 !important;
+    visibility: hidden;
+    top: 100% !important;
+    left: 100% !important;
+    z-index: -1 !important;
```

Comment:
Because you set top/left/position with propery of DOM element `dropdown[0].style.top` `dropdown[0].style.position`. z-index is additional and probably not needed took from other css class I used some time ago.